test/OFTWrapper.t.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "utils/BaseTest.sol";
+import "script/OFTWrapper.s.sol";
+
+contract OFTWrapperTest is BaseTest {
```

Comment:
I'm starting to wonder if I could make a template or base contract for these kind of layerzero testing so we have been using a similar codebase for 3 times now.